### ISSUE TITLE
remove fetch command

### ISSUE THREAD

**Issue by richardjgowers** [OpenFreeEnergy/ExampleNotebooks#21]
In the showcase notebook there's a `mda.fetch_mmtf` command, but fetching from rcsb can sometimes be problematic with firewalls.  Better to download and include the mmtf locally in the repo.

**Comment by IAlibay** [comment]
Yeah we can just create the universe from [181L_mod.pdb](https://github.com/OpenFreeEnergy/ExampleNotebooks/blob/master/openmm-rbfe/inputs/181L_mod.pdb)
